core/services/ocr2/plugins/ccip/internal/ccipdata/offramp_reader.go

```diff
@@ -19,45 +19,20 @@ const (
 	ManuallyExecute = "manuallyExecute"
 )
 
-// ExecOffchainConfig Do not change the JSON format of this struct without consulting with the RDD people first.
+// An ExecOffchainConfig specifies configuration for nodes executing committed messages.
 type ExecOffchainConfig struct {
```

Comment:
Nit: Go convention/doc comment rules dictate that comments start with the component name, so

```suggestion
// ExecOffchainConfig specifies configuration for nodes executing committed messages.
type ExecOffchainConfig struct {
```

Reply:
I followed the [Go Doc recommendation for types](https://tip.golang.org/doc/comment#type):

> A type’s doc comment should explain what each instance of that type represents or provides.
> `// A Reader serves content from a ZIP archive.`

I can remove the indefinite article if it's more consistent with the rest of the code base.

Reply:
Oh, looks like articles could be present or absent! Didn't know that. I think it's fine, as long as it matches the Go Doc rules 😄